libraries/cloudharness-common/cloudharness/events/client.py

```diff
@@ -41,16 +41,29 @@ def create_topic(self):
                                         client_id=self.client_id)
         # ## Create topic
 
-        topic_list = [NewTopic(name=self.topic_id, num_partitions=1, replication_factor=1)]
+        topic_list = []
+        for topic in [NewTopic(name=self.topic_id, num_partitions=1, replication_factor=1)]:
```

Comment:
Why use this fake list with one topic?

Reply:
kafka client create_topics supports creation of multiple topics at once

Reply:
refactored it